tests/app/questionnaire/conftest.py

```diff
@@ -831,5 +1059,49 @@ def mock_schema():
 
 
 @pytest.fixture
-def placeholder_transform():
-    return PlaceholderTransforms(language="en")
+def placeholder_transform(mock_schema, mock_renderer):
+    return PlaceholderTransforms(
+        language="en", schema=mock_schema, renderer=mock_renderer
+    )
+
+
+@pytest.fixture
+def schema_placeholder_renderer(labels_schema_with_placeholders):
+    schema = QuestionnaireSchema(labels_schema_with_placeholders)
+    answer_store = AnswerStore(
+        [
+            {"answer_id": "mandatory-radio-answer", "value": "{body_parts}"},
+            {"answer_id": "mandatory-checkbox-answer", "value": ["Body"]},
+        ]
+    )
+    renderer = PlaceholderRenderer(
+        language="en",
+        answer_store=answer_store,
+        list_store=ListStore(),
+        metadata=ImmutableDict({}),
+        response_metadata={},
+        schema=schema,
+    )
+    return schema, renderer
```

Comment:
On second thought, we should remove `labels_schema_with_placeholders` fixture schema which is essentially a duplicate of the actual test schema and instead just load it in via 
```py
load_schema_from_name("test_option_label_from_value")`
````
Then that fixture should be named explicitly to something like `option_label_from_value_schema`. If you do this approach, be sure to update the unit test asserts, as the label/value is currently mismatching.

The other option is to just mock the schema methods like `get_answers_by_answer_id`. Either way, we should look to prevent the duplication.